site2/docs/client-libraries.md

```diff
@@ -4,42 +4,74 @@ title: Pulsar client libraries
 
 ## Third-party clients
 
 Besides the official released clients, multiple projects on developing Pulsar clients are available in different languages.
 
-> If you have developed a new Pulsar client, feel free to submit a pull request and add your client to the list below.
-
-| Language | Project | Maintainer | License | Description |
-|----------|---------|------------|---------|-------------|
-| Go | [pulsar-client-go](https://github.com/Comcast/pulsar-client-go) | [Comcast](https://github.com/Comcast) | [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) | A native golang client |
-| Go | [go-pulsar](https://github.com/t2y/go-pulsar) | [t2y](https://github.com/t2y) | [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) | 
-| Haskell | [supernova](https://github.com/cr-org/supernova) | [Chatroulette](https://github.com/cr-org) | [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) | Native Pulsar client for Haskell | 
-| Scala | [neutron](https://github.com/cr-org/neutron) | [Chatroulette](https://github.com/cr-org) | [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) | Purely functional Apache Pulsar client for Scala built on top of Fs2 |
-| Scala | [pulsar4s](https://github.com/sksamuel/pulsar4s) | [sksamuel](https://github.com/sksamuel) | [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) | Idomatic, typesafe, and reactive Scala client for Apache Pulsar |
-| Rust | [pulsar-rs](https://github.com/wyyerd/pulsar-rs) | [Wyyerd Group](https://github.com/wyyerd) | [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) | Future-based Rust bindings for Apache Pulsar |
-| .NET | [pulsar-client-dotnet](https://github.com/fsharplang-ru/pulsar-client-dotnet) | [Lanayx](https://github.com/Lanayx) | [![GitHub](https://img.shields.io/badge/license-MIT-green.svg)](https://opensource.org/licenses/MIT) | Native .NET client for C#/F#/VB |
-| Node.js | [pulsar-flex](https://github.com/ayeo-flex-org/pulsar-flex) | [Daniel Sinai](https://github.com/danielsinai), [Ron Farkash](https://github.com/ronfarkash), [Gal Rosenberg](https://github.com/galrose)| [![GitHub](https://img.shields.io/badge/license-MIT-green.svg)](https://opensource.org/licenses/MIT) | Native Nodejs client |
-| PHP | [pulsar-client-php](https://github.com/ikilobyte/pulsar-client-php) | [ikilobyte](https://github.com/ikilobyte) | [![GitHub](https://img.shields.io/badge/license-MIT-green.svg)](https://opensource.org/licenses/MIT) | Native PHP client |
+> Want your repository listed here? Just submit a PR to the [pulsar repository](https://github.com/apache/pulsar/edit/master/site2/docs/client-libraries.md).
+
+### C#
```

Comment:
How about calling it `.Net`?   And for the official client as well. 
`pulsar-client-dotnet` is written in F# and the official one `pulsar-dotpulsar` is written in C#. But they are compatible with all .Net languages such as C#/F#/VB.

Reply:
I was thinking about whether `.NET` is a language. [Redis client page](https://redis.io/docs/clients/) list `.NET` language family as `C#`. I'm unsure what's the conventions of C# community.

cc @blankensteiner @Lanayx

Reply:
Hi @tisonkun - it would make sense to list C# (DotPulsar) and F# as separate languages even though they share the .NET runtime.

Reply:
@blankensteiner do you mean that we list pulsar-dotpulsar twice, once under the C# category, and the other under F# category?

Reply:
@tisonkun Many languages support calling assemblies written in other languages. So, if you want to list all languages and list which clients can be used from the languages, then you would need the C# section to also include the C/C++ client and F# client since that is technically possible. Therefore it is easier to use the language to tell what language the client is written in. Then people using F# can browse for an F# client, but also know that they can use a C# client if they want (and the other way around).

Reply:
@blankensteiner I get your point. Will keep the content as is.

I think that's why [Redis client page](https://redis.io/docs/clients/) list `.NET` language family as `C#`.

If @Lanayx has a different opinion to list `pulsar-client-dotnet` under F# as it's written in F#, I'll make the change then.

Reply:
Both Kafka and RabbitMQ refer to client as .NET clients
https://www.rabbitmq.com/devtools.html
https://docs.confluent.io/platform/current/clients/index.html

Reply:
Fixed at https://github.com/apache/pulsar/pull/17010/commits/e5c403f8e7f52b84446cc0fef392dfbe58d724b3.